03/app.js

```diff
@@ -0,0 +1,5 @@
+const allBtn = document.querySelectorAll('button');
+const handleClick = function (e) {
+    console.log(e.target.innerText = 'clicked');
+};
+allBtn.forEach(function (el) { el.addEventListener('click', handleClick, { once: true })});
```

Comment:
Dla większej czytelności polecam pisać w wielu liniach ;)

PS. Jakbyś napisał inaczej to rozwiązanie jeśli chciałbyś wspierać też IE? (zobacz, że on opcji `once` nie wspiera: https://caniuse.com/once-event-listener)